sources/packages/web/src/components/generic/DetailHeader.vue

```diff
@@ -0,0 +1,25 @@
+<template>
+  <span v-for="(headerName, index) in Object.keys(headerMap)" :key="headerName">
```

Comment:
you can also use Object.entries() 
![image](https://user-images.githubusercontent.com/77353155/191581437-f3fc29d2-d8c6-4148-aaa0-4b629070c2d2.png)

Reply:
Got it. But just trying to understand what could be the benefit of using Object.entries over Object.keys

Reply:
I see that there is a benefit of not needing to refer the object by key (obj[key]) here. I will change it.